eZ/Publish/Core/Persistence/Legacy/Content/Location/Gateway/DoctrineDatabase.php

```diff
@@ -631,6 +657,30 @@ function ($pathString) use ($query, $handler) {
         $query->prepare()->execute();
     }
 
+    private function buildHiddenSubtreeQuery(string $selectExpr): QueryBuilder
```

Comment:
`$selectExpr` is always `'node_id'`. Given the method is private, atm it does not bring value. If it's required in the future to parametrize select expression, then it would be safe to change it once needed.

Reply:
Correct, ATM the parameter have no purpose here. I can remove it if consistency between ezpublish-kernel and ezplatform-kernel has less value

Reply:
Removed param in https://github.com/ezsystems/ezpublish-kernel/pull/3142/commits/66ee6a667394b4bb962d330baef104e4e07ef042